website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -0,0 +1,491 @@
+      "registry-module": {
+        "data": {
+          "id": "mod-2aaFrmRPZs2N9epr",
+          "type": "registry-module"
+        }
+      },
+      "variable-options": {
+        "data": [
+          {
+            "type": "variable-options",
+            "attributes": {
+              "variable-name": "amis",
+              "variable-type": "string",
+              "options": [
+                "ami-1",
+                "ami-2",
+                "ami-3"
+              ]
+            }
+          },
+          {
+            "type": "variable-options",
+            "attributes": {
+              "variable-name": "region",
+              "variable-type": "string",
+              "options": [
+                "eu-north-1",
+                "us-east-2",
+                "us-west-1"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/organizations/hashicorp/no-code-modules
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "nocode-9HE91XDNY3faePn2",
+    "type": "no-code-modules",
+    "attributes": {
+      "enabled": true,
+      "version-pin": "1.0.1"
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        },
+        "links": {
+          "related": "/api/v2/organizations/hashicorp"
+        }
+      },
+      "registry-module": {
+        "data": {
+          "id": "mod-2aaFrmRPZs2N9epr",
+          "type": "registry-modules"
+        },
+        "links": {
+          "related": "/api/v2/registry-modules/mod-2aaFrmRPZs2N9epr"
+        }
+      },
+      "variable-options": {
+        "data": [
+          {
+            "id": "ncvaropt-fcHDfnZ1EGdRzFNC",
+            "type": "variable-options"
+          },
+          {
+            "id": "ncvaropt-dZMfdh9KBcwFjyv2",
+            "type": "variable-options"
+          }
+        ]
+      }
+    },
+    "links": {
+      "self": "/api/v2/no-code-modules/nocode-9HE91XDNY3faePn2"
+    }
+  }
+}
+```
+
+## Update No-Code Provisioning settings for a module.
+
+`PATCH /no-code-modules/:id`
+
+Parameter | Description
+--- | ---
+`:id` | The unique identifier for a No-Code-Allowed module.
+
+Use this API to update the settings for No-Code Provisioning of a module. You can use this API to:
+
+- Enable or disable No-Code Provisioning using this module
+- Make adjustments to the set of allowed variable values
+- Change the version of the module used during provisioning
+- Change the module that is used during provisioning
+
+The unique identifier for the No-Code-Allowed module was returned in the response to [the API call that initially enabled the module for No-Code Provisioning](#allow-no-code-provisioning-of-a-module-for-an organization).
```

Comment:
It feels like this sentence is a bit out of place? As in it should be closer to the table above? Also, maybe we can remove this sentence all together in favor of adding this link to the **Description** column above?

Reply:
```suggestion
```

I'd suggest we just remove it? I don't know if the link is even worth trying to squeeze into the table... it perhaps just doesn't need to be said?

Reply:
```suggestion
The [API call that enables for no-code provisioning for a module](#allow-no-code-provisioning-of-a-module-within-an-organization) returns that module's unique identifier.
```